frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -43,6 +45,31 @@ describe("collection permissions", () => {
             });
           });
         });
+
+        onlyOn(permission === "view", () => {
+          beforeEach(() => {
+            cy.signInAsAdmin();
+            // Give "collection" group READ-ONLY permission for root collection
+            cy.updateCollectionGraph({
+              [COLLECTION_GROUP]: { root: "read" },
+            });
+          });
+
+          userGroup.forEach(user => {
+            beforeEach(() => {
+              cy.signIn(user);
+            });
```

Comment:
This part here is problematic and I have to find a solution for it. Basically, `beforeEeach` hook execute N times for N-number of users before each test. That creates something like this (screenshot from some other test), which is definitely a bug:
![image](https://user-images.githubusercontent.com/31325167/111685446-38538880-8828-11eb-8187-8dfff38658f3.png)

OTOH, you can't use Cypress commands outside the running test OR a `before/beforeEeach` hook. This is a major blocker for this approach.